Winget-AutoUpdate/user-run.ps1

```diff
@@ -0,0 +1,131 @@
+<#
+.SYNOPSIS
+Handle user interaction from shortcuts and show a Toast
+
+.DESCRIPTION
+Act on shortcut run (DEFAULT: Check for updated Apps)
+
+.PARAMETER Logs
+Open the Log file from Winget-AutoUpdate installation location
+
+.PARAMETER Help
+Open the Web Help page
+https://github.com/Romanitho/Winget-AutoUpdate
+
+.EXAMPLE
+.\user-run.ps1 -Logs
+
+#>
+
+[CmdletBinding()]
+param(
+	[Parameter(Mandatory=$False)] [Switch] $Logs = $false,
+	[Parameter(Mandatory=$False)] [Switch] $Help = $false
+)
+
+function Show-Toast ($Title, $Message, $MessageType, $Balise, $OnClickAction) {
+
+	#Prepare OnClickAction (if set)
+	if ($OnClickAction){
+		$ToastOnClickAction = "activationType='protocol' launch='$OnClickAction'"
+	}
+
+	#Add XML variables
+	[xml]$ToastTemplate = @"
+<toast $ToastOnClickAction>
+<visual>
+    <binding template="ToastImageAndText03">
+        <text id="1">$Title</text>
+        <text id="2">$Message</text>
+        <image id="1" src="$PSScriptRoot\icons\$MessageType.png" />
+    </binding>
+</visual>
+<tag>$Balise</tag>
+</toast>
+"@
+	#Load Assemblies
+	[Windows.UI.Notifications.ToastNotificationManager, Windows.UI.Notifications, ContentType = WindowsRuntime] | Out-Null
+	[Windows.Data.Xml.Dom.XmlDocument, Windows.Data.Xml.Dom.XmlDocument, ContentType = WindowsRuntime] | Out-Null
+
+	#Prepare XML
+	$ToastXml = [Windows.Data.Xml.Dom.XmlDocument]::New()
+	$ToastXml.LoadXml($ToastTemplate.OuterXml)
+
+	#Specify Launcher App ID
+	$LauncherID = "Windows.SystemToast.Winget.Notification"
+
+	#Prepare and Create Toast
+	$ToastMessage = [Windows.UI.Notifications.ToastNotification]::New($ToastXml)
+	$ToastMessage.Tag = $ToastTemplate.toast.tag
+	[Windows.UI.Notifications.ToastNotificationManager]::CreateToastNotifier($LauncherID).Show($ToastMessage)
+}
+
+Function Get-ToastLocale {
+    
```

Comment:
Then I need to rewrite the function and load just that (howto?)  into the **standalone** user-run.ps1...
...I've added a test for the new strings...

Reply:
I think it's better to keep the user-run.ps1 as a standalone!

Reply:
I understand the fact to keep it as standalone. But we might reuse things we already created, even if we need to adapt the existing function. This way, if we need to changes few things, it can be done on one place only. I think it's better like this. Because I might change the functions a bit.

Reply:
...? so **standalone** or edit/adapt your functions to accommodate a call from user-run.ps1 and initiate the log system.
Do you mean the later?